src/__swaps__/screens/Swap/providers/swap-provider.tsx

```diff
@@ -305,14 +304,14 @@ export const SwapProvider = ({ children }: SwapProviderProps) => {
         userAssetsQueryKey({
           address: parameters.quote.from,
           currency: nativeCurrency,
-          connectedToHardhat,
+          testnetMode: connectedToHardhat,
```

Comment:
this invalidation is for the old user assets stuff so can probably be removed

Reply:
everything has been swapped out 1:1 old -> new so shouldn't it be kept?

Reply:
yeah don't think we need it if nothing is referencing the old data